receiver/awsxrayreceiver/internal/translator/cause.go

```diff
@@ -68,8 +68,7 @@ func addCause(seg *awsxray.Segment, span ptrace.Span) {
 			attrs := evt.Attributes()
 			attrs.EnsureCapacity(8)
 
-			// ID is a required field
-			attrs.PutStr(awsxray.AWSXrayExceptionIDAttribute, *excp.ID)
+			addString(excp.ID, awsxray.AWSXrayExceptionIDAttribute, attrs)
```

Comment:
Does this change merit a changelog entry as it appears to be a change in behaviour?

Reply:
The awsxrayreceiver is for receiving data from AWS X-Ray SDKs. 
https://github.com/aws/aws-xray-sdk-python
https://github.com/aws/aws-xray-sdk-node
https://github.com/aws/aws-xray-sdk-java
https://github.com/aws/aws-xray-sdk-dotnet
https://github.com/aws/aws-xray-sdk-ruby

None of them uses Exception.Id so it won't break X-Ray SDK users.

Reply:
I'm not a codeowner of this component but it feels like a changelog is warranted, just to be safe.